### ISSUE TITLE
coqbot minimize should be able to minimize .v files

### ISSUE THREAD

**Issue by JasonGross** [coq/bot#256]
`@coqbot minimize` treats its argument as a bash file.  This is fully general, bit it would be nice to support Coq files for convenience.  Maybe we should default to Coq file, and only treat it as bash if the user writes ` ```bash ` or ` ```sh `?

**Comment by JasonGross** [comment]
That should also be pretty easy, we can just recognize the text that GitHub inserts for drag-and-drop and have some default script for downloading links

**Comment by Zimmi48** [comment]
The syntax could be: `@coqbot minimize file=[](file.v.gz)` or:

~~~
@coqbot minimize coq=8.16 archive=[](archive.zip)

```sh
unzip archive.zip
cd foo
make
```

**Comment by JasonGross** [comment]
I don't think this is a convenience worth providing.  When dragging and dropping a file like [foo.v.tar.gz](https://github.com/coq/bot/files/10086338/foo.v.tar.gz) to github, there is already a url in `[foo.v.tar.gz](https://github.com/coq/bot/files/10086338/foo.v.tar.gz)`.  The user can then just move the link upfront or something and make the script
```sh
wget https://github.com/coq/bot/files/10086338/foo.v.tar.gz
tar -xzvf foo.v.tar.gz
coqc -q foo.v
```
or whatever.  I think instead we should have a script that can do some smart guessing, like:
- if the link is a `.txt` file rename it to `bug.v` and invoke `coqc -q` on it
- if it's an archive, extract it, and then:
  - if there's only one .v file, invoke `coqc -q` on it
  - if there's a `Makefile` (or similar), `cd` to the top-most directory containing such and run `make` (alternatively just try `make` in every subdirectory or something)
  - otherwise try `coqc -q` on each `.v` file
 
And then we should recognize a form that is `@coqbot minimize <options> [text](link)` where there is no ` ``` ` in `<options>` nor after the link.

**Comment by Zimmi48** [comment]
@JasonGross The documentation of this feature, currently located at https://github.com/coq/coq/wiki/Coqbot-minimize-feature, should probably be updated.

BTW, I don't think that the current location for the documentation of this feature is good, because it's hardly discoverable. Probably, it should be a specific file in the bot repo, that would be linked from both the bot README *and* the contributing guide of Coq (currently, this section https://github.com/coq/coq/blob/master/CONTRIBUTING.md#reporting-a-bug-requesting-an-enhancement references the `find-bug.py` utility but not the coqbot feature).

**Comment by JasonGross** [comment]
I've updated the documentation a bit, I'll make an issue so we remember to move the doc at some point
